### PR TITLE
Remove appShell as we dont have one

### LINKED ISSUE #1223: Remove `appShell` from OfflinePlugin
**Is your feature request related to a problem? Please describe.**
Our Service Worker `appShell` is `/news/articles`: https://github.com/bbc/simorgh/blob/latest/webpack.config.client.js#L89

Does that work?

The docs suggest it should point to a HTML-only route, e.g. `/app-shell.html`: https://github.com/NekR/offline-plugin/blob/master/docs/app-shell.md
 
The only HTML this is going to cache is `No route was found for /news/articles/index.html.` !!

It’s intended for PWA client-side renders (on _first_ page load). The default value is `null`, so we should just remove the option.

**Describe the solution you'd like**
A one-line PR which removes `appShell` from the webpack config.

**Describe alternatives you've considered**
Leaving it - we're likely to kill off the whole thing in #1217 anyway. But one day when we _do_ come to reimplementing the offline plugin, it would be good to have a more 'correct' basis to start from.

**Testing notes**
[Tester to complete]

Dev insight: Regression test only.

**Additional context**
N/a

- [x] Initially labelled with ["Refinement needed"](https://github.com/bbc/simorgh/labels/Refinement%20Needed)

### PR DESCRIPTION
**Pull request by dr3**
Resolves #1223

**Overall change:** Remove `appShell` from service worker generation as the page doesnt exist.

**Code changes:**

- Remove `appShell` from webpack client config

**Testing**
- Regression to check nothing is broken, This should have zero affect

---

- [x] I have assigned myself to this PR and the corresponding issues
- [ ] Tests added for new features
- [ ] Test engineer approval
- [ ] I have followed [the merging checklist](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/MERGE_PROCESS.md) and this is ready to merge.

### CODE REVIEW

**Comment by dr3** [comment]
@ChrisBAshton to confirm but I don't think this has any visual change atm, I don't think this errored

**Review by ChrisBAshton — DISMISSED** [review]

**Review by pjlee11 — DISMISSED** [review]

**Comment by ChrisBAshton** [comment]
Failing on Lighthouse because it lowers our PWA rating. Drew to investigate solutions.

**Review by bcmn — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -35,7 +35,7 @@
     "test:e2e:storybook": "npm run killStorybook && run-p --race storybook cypress:storybook",
     "test:e2e:storybook:ci": "run-p --race storybook cypress:storybook",
     "test:e2e:storybook:interactive": "npm run killStorybook && run-p --race storybook cypress:storybook:interactive",
-    "test:lighthouse": "lighthouse-ci http://localhost:7080/news/articles/c9rpqy7pmypo --performance=80 --pwa=92 --accessibility=100 --best-practices=93 --seo=80 && lighthouse-ci http://localhost:7080/news/articles/c85pqyj5m2ko --performance=80 --pwa=92 --accessibility=100 --best-practices=93 --seo=80",
+    "test:lighthouse": "lighthouse-ci http://localhost:7080/news/articles/c9rpqy7pmypo --performance=80 --pwa=62 --accessibility=100 --best-practices=93 --seo=80 && lighthouse-ci http://localhost:7080/news/articles/c85pqyj5m2ko --performance=80 --pwa=62 --accessibility=100 --best-practices=93 --seo=80",
```
Comment (sareh) [inline package.json:38]: It's a shame that we're decreasing our PWA rating by 30 points, but we should be able to address this in a future ticket: https://github.com/bbc/simorgh/issues/755

**Review by sareh — APPROVED** [review]

**Comment by jamesbrumpton** [comment]
LGTM 👍

**Pull request merged**